docs/clients/service-providers.md

```diff
@@ -0,0 +1,103 @@
+# Integrate a Cosmos-SDK based blockchain as a Service Provider
+
+We define 'service providers' as entities providing services for end-users that involve some form of interaction with a Cosmos-SDK based blockchain (this includes the Cosmos Hub). More specifically, this document will be focused around interactions with tokens.
+
+This section does not concern wallet builders that want to provide Light-Client functionalities. Service providers are expected to act as trusted point of contact to the blockchain for their end-users. 
+
+## High-level description of the architecture
+
+There are three main pieces to consider:
+
+- Full-nodes: To interact with the blockchain. 
+- Rest Server: This acts as a relayer for HTTP calls.
+- Rest API: Define available endpoints for the Rest Server.
+
+## Running a Full-Node
+
+### Installation and configuration
+
+We will describe the steps to run and interract with a full-node for the Cosmos Hub. For other SDK-based blockchain, the process should be similar. 
+
+First, you need to [install the software](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/getting-started/installation.md).
+
+Then, you can start [running a full-node](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/getting-started/full-node.md).
+
+### Command-Line interface
+
+Next you will find a few useful CLI commands to interact with the Full-Node.
```

Comment:
should you include a command like `gaiacli help` or something?

Reply:
yep probably